pymc3/distributions/bound.py

```diff
@@ -266,25 +184,37 @@ def __call__(self, name, *args, **kwargs):
                 "with the cumulative probability function."
             )
 
-        transform = kwargs.pop("transform", "infer")
-        if issubclass(self.distribution, Continuous):
-            return _ContinuousBounded(
-                name, self.distribution, self.lower, self.upper, transform, *args, **kwargs
-            )
-        elif issubclass(self.distribution, Discrete):
-            return _DiscreteBounded(
-                name, self.distribution, self.lower, self.upper, transform, *args, **kwargs
+        if not isinstance(distribution, TensorVariable):
+            raise ValueError(
+                "Passing uninstantiated classes of distributions to"
+                "Bound is no longer supported. Please pass the"
+                "corresponding distribution after instantiation."
             )
-        else:
-            raise ValueError("Distribution is neither continuous nor discrete.")
 
-    def dist(self, *args, **kwargs):
-        if issubclass(self.distribution, Continuous):
-            return _ContinuousBounded.dist(
-                self.distribution, self.lower, self.upper, *args, **kwargs
-            )
+        mv_var_list = (
+            MvNormalRV,
+            MvStudentTRV,
+        )
 
-        elif issubclass(self.distribution, Discrete):
-            return _DiscreteBounded.dist(self.distribution, self.lower, self.upper, *args, **kwargs)
-        else:
-            raise ValueError("Distribution is neither continuous nor discrete.")
+        if isinstance(distribution.owner.op, mv_var_list):
+            raise NotImplementedError("Bounding of MultiVariate RVs is not yet supported.")
```

Comment:
There should be a better way of doing this rather than a long list of Multivariate RVs